kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml

```diff
@@ -184,4 +185,141 @@
       </property>
     </bean>
 
+    <bean id="documentCreationJob" parent="unscheduledJobDescriptor">
+      <property name="steps">
+        <list>
+          <ref bean="transactionFilesValidationStep"/>
+          <ref bean="transactionFilesConsolidateStep"/>
+          <ref bean="documentCreationStep"/>
+        </list>
+      </property>
+    </bean>
+
+
+    <bean id="transactionFilesValidationStep" class="edu.arizona.kfs.fp.batch.TransactionFilesValidationStep" parent="step">
+      <property name="bankTransactionsLoadService">
+        <ref bean="bankTransactionsLoadService"/>
+      </property>
+      <property name="batchInputFileType" ref="csvBankTransactionsInputFileType"/>
+    </bean>
+
+    <bean id="transactionFilesConsolidateStep" class="edu.arizona.kfs.fp.batch.TransactionFilesConsolidateStep" parent="step">
+      <property name="bankTransactionsLoadService">
+        <ref bean="bankTransactionsLoadService"/>
+      </property>
+      <property name="batchInputFileType" ref="csvBankTransactionsInputFileType"/>
+    </bean>
+
+    <bean id="documentCreationStep" class="edu.arizona.kfs.fp.batch.DocumentCreationStep" parent="step">
+      <property name="transactionPostingService">
+        <ref bean="transactionPostingService"/>
+      </property>
+      <property name="batchInputFileType" ref="csvBankTransactionsInputFileType"/>
+    </bean>
+
+
+  <bean id="bankTransactionsLoadService" class="edu.arizona.kfs.fp.batch.service.impl.BankTransactionsLoadServiceImpl" scope="prototype">
+    <property name="bankParametersAccessService">
+      <ref bean="bankParametersAccessService"/>
+    </property>
+    <property name="batchInputFileService">
+      <ref bean="batchInputFileService"/>
+    </property>
+    <property name="businessObjectService">
+      <ref bean="businessObjectService"/>
+    </property>
+    <property name="batchInputFileType">
+        <ref bean="csvBankTransactionsInputFileType" />
+    </property>
+    <property name="bankTransactionsValidatedFileType">
+      <ref bean="csvBankTransactionsValidatedFileType" />
+    </property>
+    <property name="transactionPostingDao">
+      <ref bean="transactionPostingDao" />
+    </property>
+  </bean>
+
+
+  <bean id="transactionPostingService" class="edu.arizona.kfs.fp.batch.service.impl.TransactionPostingServiceImpl">
+    <!--property name="batchInputFileService">
```

Comment:
I'm planning to add it back in the next iteration for the documentCreation Step. I'll delete it for now.